tests/Fixture/BakeTemplateRolesFixture.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Created by javier
+ * Date: 31/01/16
+ * Time: 10:59
+ */
```

Comment:
These should be the standard file header used across the project. I can take care of that.